### PR TITLE
fix(integrations): ensure non-typed documents get passed along to the…

### LINKED ISSUE #990: Cannot read property 'initCfg' of undefined
So here I just created a quick oracle EBS to twitter integration and then added a data mapper step.  The data mapper fails to load with the below error:

capture

@akieling @igarashitm FYI...

### PR DESCRIPTION
**Pull request by gashcrumb**
… data mapper

fixes #990

### CODE REVIEW

File: src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts [hunk 1]
```diff
@@ -129,45 +129,30 @@ export class DataMapperHostComponent extends FlowPage
...
 
   createDocumentDefinition(dataShape: DataShape, isSource: boolean = false) {
-    if (!dataShape || !dataShape.type || !dataShape.kind) {
+    if (!dataShape || !dataShape.kind) {
       // skip
       return;
     }
     const type = dataShape.type;
     const kind = dataShape.kind;
-    // avoid passing duplicates off to the data mapper
```
Comment (igarashitm) [inline src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts]: I don't know the background though, this duplicate check is no longer required?

File: src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts [hunk 2]
```diff
@@ -129,45 +129,30 @@ export class DataMapperHostComponent extends FlowPage
...
-        return;
-      }
-      this.sourceDocTypes.push(type);
-    } else {
-      if (this.targetDocTypes.find(d => d === type)) {
-        return;
-      }
-      this.targetDocTypes.push(type);
-    }
-    log.infoc(
```
Comment (igarashitm) [inline src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts]: I think we want this log at least?

File: src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts (hunk 1 above)
Comment (gashcrumb) [reply src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts]: I don't think so, previously there were issues where the component was getting initialized more than once, those should be sorted.

File: src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts (hunk 2 above)
Comment (gashcrumb) [reply src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts]: I kinda find the logs noisy enough as it is :-)

Comment (igarashitm) [reply src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts]: right? if you say so :)

**Pull request merged**